static/js/containers/CreatePostPage.js

```diff
@@ -17,12 +18,16 @@ import type { Match } from "react-router"
 type CreatePostPageProps = {
   match: Match,
   dispatch: Dispatch,
-  postForm: FormValue,
+  postForm: ?FormValue,
   channel: Channel,
   channels: RestState<Array<Channel>>,
   history: Object
 }
 
+const CREATE_POST_KEY = "post:new"
+const CREATE_POST_PAYLOAD = { key: CREATE_POST_KEY }
+const getForm = R.prop(CREATE_POST_KEY)
```

Comment:
A few nitpicks that apply equally to this container and the channel container: 

1. Should we use `"post:new"` for the key name? If I'm understanding correctly, the key is just being used to identify the correct form in the state, which isn't necessarily 'new'. Maybe just `"post_form"` is more on the mark.
1. I think `CREATE_POST_KEY` should be changed to `POST_FORM_KEY`, and `POST_FORM_KEY_PAYLOAD` or `POST_FORM_IDENTIFIER` might be a little clearer (IMO) than `CREATE_POST_PAYLOAD` (`_PAYLOAD` makes me think that this is the entire payload you will be sending to the action)

Also, would it make sense to stick these variables in `static/js/lib/posts.js`? You could also create a `POST_FORM_INITIAL_STATE` object there and use that as the payload to `formBeginEdit` in `componentWillMount`. I'll leave that to you